zubhub_frontend/zubhub/src/assets/js/styles/components/button/buttonStyles.js

```diff
@@ -7,6 +7,12 @@ const styles = theme => ({
       backgroundColor: '#03848C',
     },
   },
+  primaryButtonStyle2:{
```

Comment:
If using vscode, please use the plugin prettier.  If it works for you please set it as default formatter and set format on save. 
or manually run the code through cli/vscode to format the code

Reply:
would you please specify what is not well formatted because I'm actually using prettier and it is enabled, I may have to change it's configuration.